Index: arch/arm/mach-omap2/board-flash.c
===================================================================
--- arch/arm/mach-omap2/board-flash.c	(revision 7)
+++ arch/arm/mach-omap2/board-flash.c	(working copy)
@@ -144,6 +144,7 @@
 	omap_nand_data.nr_parts		= nr_parts;
 	omap_nand_data.devsize		= nand_type;
 	omap_nand_data.gpmc_t		= gpmc_t;
+	omap_nand_data.dev_ready        = true;/* Use RDY/BSY pin instead of polling in nandflash driver. Added by MYIR */
 
 	return &omap_nand_data;
 }
